pupil_src/shared_modules/marker_tracker_3d/camera_localizer.py

```diff
@@ -0,0 +1,40 @@
+from marker_tracker_3d import math
+from marker_tracker_3d.localization import Localization
+
+
+class CameraLocalizer:
+    def __init__(self, storage):
+        self.storage = storage
+
+        self.localization = Localization(
+            self.storage.camera_model, self.storage.marker_model
+        )
+        self.min_number_of_markers_per_frame_for_loc = 2
+
+    def update(self):
```

Comment:
Good job splitting all of this into separate functions! This gives a great overview of what is happening in this function! :+1: